### ISSUE TITLE
Exported bookmark lists with accented characters get renamed to generic names

### ISSUE THREAD

**Issue by nonobio** [organicmaps/organicmaps#8201]
**Describe the issue**
When exporting multiple bookmark lists at once using the Organic Maps Android app, lists with names containing accented characters (such as é, ê, ù, etc.) are replaced by generic names like OrganicMaps_1, OrganicMaps_2, etc.

**Steps to reproduce**
1. Open the Organic Maps app on an Android device
2. Create multiple bookmark lists with names containing accented characters (e.g., "Café", "Montagne", "Vacances")
3. Export all bookmark lists at once

**Expected behaviour**
The exported bookmark lists should retain their original names, including any accented characters.

**Screenshots**
![image](https://github.com/organicmaps/organicmaps/assets/7425799/9361c6c4-687b-4660-ac6d-2c1d742ecaaa)

**System information:**
 - Operating system and its version: [Android 10]
 - Organic Maps version: [2024.03.31-8-Google]
 - Device Model: [Motorola Moto G7 Plus]

**Additional context**
This function is great, I hadn't noticed it and I was exporting my bookmark lists one by one.
Until this bug is eventually resolved, I am careful that there are no accents in the names of my lists.
Apparently, there is no problem with the accents of bookmark names.

Thanks :)

**Comment by biodranik** [comment]
The main issue is that using non-ASCII characters in file names inside a .zip file may not work properly on all platforms. Did anyone make a research about the current state of this issue? What is the most reliable way to generate compatible .kmz files? (it's a kml inside a .zip)

**Comment by nonobio** [comment]
Thank you for your answer. In the meantime I saw that I had to make a duplicate of https://github.com/organicmaps/organicmaps/issues/7453

It's interesting to know that the kmz is a simple zip, which allows you to extract the kml files separately if necessary.
I wanted to edit the kmz OM file opened in Google Earth, I then saved the location in kmz, but it does not open in OM. When working on kml files it seems to be less prone to errors but I often have problems with kml file names and/or the presence of an unnecessary subfolder. Anyway, do you have any recommendations for editing kmz or kml files between OM and Google Earth?

**Comment by krozhdestvenski** [comment]
can't reproduce
<img width="641" alt="image" src="https://github.com/organicmaps/organicmaps/assets/36443562/939a352b-32b7-45b0-a366-2ab4d36dc406">

**System information**:
Operating system and its version: [Android 14]
Organic Maps version: [2024.05.09-1-FDroid-debug]
Device Model: [Google Pixel 7 API 34 (emulated)]

**Comment by nonobio** [comment]
> can't reproduce
> **System information**: Operating system and its version: [Android 14] Organic Maps version: [2024.05.09-1-FDroid-debug] Device Model: [Google Pixel 7 API 34 (emulated)]

Weird. Is it Google Earth that you used? I don't recognize him.
I edited my first post to provide more details and I also attached kmz files to test.

> please share any kmz that are not open in OM, we need to fix it.
Here is an example of a kmz file that does not open in OrganicMaps:
[Organic Maps Bookmarks and Tracks_Saved from Google Earth.zip](https://github.com/organicmaps/organicmaps/files/15374235/Organic.Maps.Bookmarks.and.Tracks_Saved.from.Google.Earth.zip)

But, I don't know if this is a bug or just normal behavior:
- I open a kmz file exported from the Export all lists function of Organic Maps (see example of files in my first post)
- I open the file in Google Earth (Windows)
- I save the file from Google Earth:
![image](https://github.com/organicmaps/organicmaps/assets/7425799/0b10b327-88ec-43cb-9c88-838150388355)
- I then try to open this new kmz file in Organic Maps but it doesn't work:
![image](https://github.com/organicmaps/organicmaps/assets/7425799/75303dba-2501-40f9-ba8e-12775db19ce4)
You will tell me, but why do that? ;) Actually, I thought I could edit this OrganicMapsBackup.kmz file in Google Earth. I just renamed a list, and I wanted to save my modification like this. But I notice that the kmz file is very small and that it only contains (by opening it in zip with 7zip) the doc.kml file (there is no longer the files folder containing the kml lists).
Afterward, I realize that I can't really modify this kmz in Google Earth: for example, I can't add places to my lists.
This is why I start by unzipping the kmz before opening it in Google Earth to make modifications, as I explain [here](https://github.com/orgs/organicmaps/discussions/8204). This seems to be the way to use but tell me again if there is a way to edit this kmz in Google Earth directly...

> Google Earth is a good tool, you may also try https://umap.openstreetmap.fr/

Thank you, I tried to import my OrganicMapsBackup.kmz file but I can't expand the contents of my OrganicMapsBackup.kmz file:![image](https://github.com/organicmaps/organicmaps/assets/7425799/9f8c771e-b92a-4634-8e34-f794d64772e0)

The I tried with a kml file, it opens but I was not very successful in finding the functions that I use in Google Earth Window, at least for this use of kml kmz management.

**Comment by biodranik** [comment]
The file that you claim is not opening contains only references to your other _local_ files, that are obviously absent in the kmz archive:

```
<?xml version="1.0" encoding="UTF-8"?>
<kml xmlns="http://www.opengis.net/kml/2.2" xmlns:gx="http://www.google.com/kml/ext/2.2" xmlns:kml="http://www.opengis.net/kml/2.2" xmlns:atom="http://www.w3.org/2005/Atom">
<Document>
    <name>Organic Maps Bookmarks and Tracks</name>
    <open>1</open>
    <NetworkLink>
        <name>Chateaux</name>
        <Link>
            <href>OrganicMapsBackup_List names without accents.kmz/files/Chateaux.kml</href>
        </Link>
    </NetworkLink>
    <NetworkLink>
        <name>Ou manger</name>
        <Link>
            <href>OrganicMapsBackup_List names without accents.kmz/files/Ou manger.kml</href>
        </Link>
    </NetworkLink>
    <NetworkLink>
        <name>Forets</name>
        <Link>
            <href>OrganicMapsBackup_List names without accents.kmz/files/Forets.kml</href>
        </Link>
    </NetworkLink>
    <NetworkLink>
        <name>Cafes</name>
        <Link>
            <href>OrganicMapsBackup_List names without accents.kmz/files/Cafes.kml</href>
        </Link>
    </NetworkLink>
    <NetworkLink>
        <name>Rivieres</name>
        <Link>
            <href>OrganicMapsBackup_List names without accents.kmz/files/Rivieres.kml</href>
        </Link>
    </NetworkLink>
</Document>
</kml>
```

**Comment by nonobio** [comment]
Yes indeed, there is no longer the files folder and the kml inside the kmz, so it must be Google Earth which is not recording everything correctly. It only seems to save the root element that I right-click on to "save"...
I don't know if this is normal behavior or a bug.

**Comment by LaurentGrenet** [comment]
Where are we today about this bug ?

On the other hand, when exporting a single track that names includes stressed characters, the created KMZ file uses the stressed characters in its name, **and it do not generate any drawback.**

So, I can't see any reason to do not apply the same in global export.
